internal/config/config.go

```diff
@@ -176,7 +176,7 @@ var configOptions = []ConfigOption{
 	},
 	{
 		Key:           "prompt",
-		Description:   "toggle interactive prompting in the terminal",
+		Description:   "toggle interactive prompting in the terminal. Overridden if GH_PROMPT_DISABLED is set",
```

Comment:
wasn't sure if this should be updated or not, I felt it might be confusing otherwise.

Reply:
We don't indicate for the other config options that they can be overridden by environment variables so I would be in favor of leaving this description be as is.